.eslintrc

```diff
@@ -70,6 +70,6 @@
     ],
     "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks
     "react-hooks/exhaustive-deps": "warn", // Checks effect dependencies
-    "require-jsdoc": "warn"
+    "require-jsdoc": "off"
```

Comment:
Why are we disabling the warning?

Reply:
I was actually wondering why we enabled it to begin with, considering that its off for the other repos. and I'm pretty sure that this was off to begin this repo with. we are not even addressing the warnings. What is the point of the having it on right now?

Reply:
leave it on warn for now, it doesn't error eslint since it's on warn
If you want to turn it off, disable it in another PR (with a ci-skip commit)

Reply:
If we want to disable it, let's have a discussion around it and not try to include it in a PR about a component.